client/components/payment-method-logos/index.tsx

```diff
@@ -0,0 +1,113 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import {
+	Visa,
+	MasterCard,
+	Amex,
+	WooPay,
+	ApplePay,
+	GooglePay,
+	Discover,
+	JCB,
+	Afterpay,
+	Affirm,
+	Klarna,
+} from './Icons';
+
+import './style.scss';
+
+const Payments = [
+	{
+		name: 'visa',
+		component: <Visa key="visa" />,
+	},
+	{
+		name: 'mastercard',
+		component: <MasterCard key="mastercard" />,
+	},
+	{
+		name: 'amex',
+		component: <Amex key="amex" />,
+	},
+	{
+		name: 'discover',
+		component: <Discover key="discover" />,
+	},
+	{
+		name: 'woopay',
+		component: <WooPay key="woopay" />,
+	},
+	{
+		name: 'applepay',
+		component: <ApplePay key="applepay" />,
+	},
+	{
+		name: 'googlepay',
+		component: <GooglePay key="googlepay" />,
+	},
+	{
+		name: 'afterpay',
+		component: <Afterpay key="afterpay" />,
+	},
+	{
+		name: 'affirm',
+		component: <Affirm key="affirm" />,
+	},
+	{
+		name: 'klarna',
+		component: <Klarna key="klarna" />,
+	},
+	{
+		name: 'jcb',
+		component: <JCB key="jcb" />,
+	},
+];
+
+export const WooPaymentMethodLogos: React.VFC< {
```

Comment:
```suggestion
export const WooPaymentMethodsLogos: React.VFC< {
```

Reply:
changed in [af009a5](https://github.com/Automattic/woocommerce-payments/pull/8977/commits/af009a5035144213ca7d93bc162bb035dd0a639d). thx